tardis/io/atom_data/base.py

```diff
@@ -164,7 +164,7 @@ def from_hdf(cls, fname=None):
                 try:
                     dataframes[name] = store[name]
                 except KeyError:
-                    logger.debug("Dataframe does not contain NAME column")
+                    logger.debug(f"Dataframe does not contain {name} column")
```

Comment:
What does this print? Is it a column header or an entire column?

Reply:
This is what it used to print
![image](https://user-images.githubusercontent.com/66117751/125472494-0a2f921b-55d2-4108-aca8-76c3f1d9828d.png)
And this is what it prints right now (after the change)
![image](https://user-images.githubusercontent.com/66117751/125472644-c9ffc7a0-7926-4046-a5ed-e37a32ea213f.png)

Reply:
Much better, thanks